site/docs/guide/context.md

```diff
@@ -83,6 +83,34 @@ bot.on("edited_message", (ctx) => {
 
 Hence, if you want to, you can forget about `ctx.message` and `ctx.channelPost` and `ctx.editedMessage` and so on and so forth, and just always use `ctx.msg` instead.
 
+## Probing via Has-checks
```

Comment:
I don't think we usually use a hyphen to connect nouns.

Reply:
Has isn't usually a noun.

Reply:
"Has" itself isn't a noun, but it is a noun in that context.

Reply:
I prefer Has Checks. The article is neither Russian to say Has-Checks, nor it is Dutch or German to say Haschecks.

Reply:
It is comparable to words like one-time or half-baked

Reply:
I'd be happy to have a different and better name if you can come up with one hehe

Reply:
One-time and half-baked are adjectives, and it is common to use a hyphen to form adjectives.

Reply:
Alright, I'll change it